cloudinit/distros/bsd.py

```diff
@@ -97,6 +97,10 @@ def package_command(self, command, args=None, pkgs=None):
             if not self.pkg_cmd_update_prefix:
                 return
             cmd = self.pkg_cmd_update_prefix
+        elif command == 'upgrade':
+            if not self.pkg_cmd_upgrade_prefix:
+                return
```

Comment:
Is it worth a log.debug message here that 'package upgrade' not supported on this distro?

Reply:
No, it may instead confused the user.